libs/ECS/include/Scene.h

```diff
@@ -0,0 +1,25 @@
+//
+// Created by Daan van Donk on 10/05/2022.
+//
+
+#ifndef PROFTAAK24_SCENE_H
+#define PROFTAAK24_SCENE_H
+
+
+#include <vector>
+#include "GameObject.h"
+
+class Scene
+{
+protected:
+    static float deltaTime;
```

Comment:
DeltaTime wordt nooit geset en zal altijd 0 zijn.

Reply:
Die feature moet nog komen. Voor nu is het goed genoeg om verder te kunnen gaan. Er zijn nog veel dingen die ECS nodig heeft maar voor nu zouden een proof of concept kunnen worden gemaakt